drivers/pwm/pwm_nrfx.c

```diff
@@ -350,6 +359,10 @@ static int pwm_nrfx_pm_control(struct device *dev,
 	(DT_NORDIC_NRF_PWM_PWM_##dev_idx##_CH##ch_idx##_INVERTED ?            \
 	 PWM_NRFX_CH_VALUE_INVERTED : PWM_NRFX_CH_VALUE_NORMAL)
 
+#define PWM_NRFX_COUNT_MODE(dev_idx)                                          \
+	(DT_NORDIC_NRF_PWM_PWM_##dev_idx##_CENTER_ALIGNED ?                   \
```

Comment:
You could use `IS_ENABLED(DT_NORDIC_NRF_PWM_PWM_##dev_idx##_CENTER_ALIGNED)` here. Then, there would be no need for adding these `#ifndef`s below.